app/controllers/law_offices_controller.rb

```diff
@@ -1,7 +1,8 @@
 class LawOfficesController < ApplicationController
   def index
     @search = params[:search]
-    @tokens = @search.split
+      @search = '' if @search == nil
+      @tokens = @search.split(",")
```

Comment:
We'll actually want to leave this as `@tokens = @search.split` (without the `(",")` part).  This will make each word the user types in its own term to search on.